tests/epyccel/modules/list_comprehension.py

```diff
@@ -0,0 +1,4 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+
+F = [False for _ in range(5)]
```

Comment:
F was the name of the variable in the old file, Is it better to change it?

Reply:
updated 378ade635b72a58c302931e7c76ddc2f16d4f3c6